### ISSUE TITLE
refineDiag_ocean_month*.py analysis scripts are broken

### ISSUE THREAD

**Issue by nikizadehgfdl** [NOAA-GFDL/MOM6-examples#200]
dev/gfdl/2017.10.19
All of refineDiag python scripts fail with KeyError: 'basin' :

```
/nbhome/oar.gfdl.ogrp-account/warsaw_201710_mom6_20171222/CM4_c96L33_am4p0_2010_glac/mom6/tools/analysis/refineDiag_ocean_month.py -b 00050101.ocean_static_no_mask_table.nc -r /vftmp/oar.gfdl.ogrp-account/pbs16366214/warsaw_201710_mom6_20171222/CM4_c96L33_am4p0_2010_glac/gfdl.ncrc4-intel16-prod-openmp/CM4_c96L33_am4p0_2010_glac_00050101/history_refineDiag/00050101.nc 00050101.ocean_month.nc
Traceback (most recent call last):
  File "/nbhome/oar.gfdl.ogrp-account/warsaw_201710_mom6_20171222/CM4_c96L33_am4p0_2010_glac/mom6/tools/analysis/refineDiag_ocean_month.py", line 249, in <module>
    run()
  File "/nbhome/oar.gfdl.ogrp-account/warsaw_201710_mom6_20171222/CM4_c96L33_am4p0_2010_glac/mom6/tools/analysis/refineDiag_ocean_month.py", line 40, in run
    main(args)
  File "/nbhome/oar.gfdl.ogrp-account/warsaw_201710_mom6_20171222/CM4_c96L33_am4p0_2010_glac/mom6/tools/analysis/refineDiag_ocean_month.py", line 69, in main
    basin_code = f_basin.variables['basin'][:]
KeyError: 'basin'
```

**Comment by nikizadehgfdl** [comment]
@adcroft  Why are these scripts called with " -b $ocean_static_file " rather than "-b $yr1.basin_codes.nc" ?
Is there a plan to put the basin codes in ocean_static_no_mask_table.nc in near future?

**Comment by adcroft** [comment]
Closed by #201
